bazelfe-bazel-wrapper/src/bep/build_events/hydrated_stream.rs

```diff
@@ -334,7 +351,7 @@ mod tests {
     #[tokio::test]
     async fn test_no_history() {
         let (tx, rx) = async_channel::unbounded();
-        let mut child_rx = pin!(HydratedInfo::build_transformer(rx));
+        let mut child_rx = std::pin::pin!(HydratedInfo::build_transformer(rx));
```

Comment:
`use std::pin::pin` gives an unused warning... but if I remove it, then it won't compile.